### DISCUSSION TITLE
Plan Types

### DISCUSSION THREAD

**Discussion by V1d1o7** [TandoorRecipes/recipes#395] [Q&A] [answered]
Since this isn't really an "issue" more of a incomprehension I opened here.
What is the purpose fo the "Plan Types' when doing the meal planning? I don't fully understand the idea behind it and the doc site doesn't mention anything.

**Comment by V1d1o7** [comment]
On the same note, I understand "Books" is to divide up recipes and sort them, but why not have it show what book it is in on search like tags do and why not have it an option to add to that book while editing the recipe vs after. I know you can have your "homepage" set to books which does help a little.

**Reply by pavelsolar** [reply]
This feature is work in progress I belive. I also would like to expand it a bit more (possibility to share, print, add sections etc.), but for now I'm OK with it. Overall it's a wonderful project.

**Comment by pavelsolar** [comment]
> Since this isn't really an "issue" more of a incomprehension I opened here.
> What is the purpose fo the "Plan Types' when doing the meal planning? I don't fully understand the idea behind it and the doc site doesn't mention anything.

The way I see this feature - You can create more Plan types for different family members (kids, vegans, to give an example). Or you can separate differrent meal type of the day (breakfast, dinner, lunch ...). If you plant to use it only for yourself, create just one Plan type and you're done.

**Comment by vabene1111** [comment, marked as answer]
Thanks for all the Feedback. I would love to hear your ideas on how to improve books, but as you say, to be completely honest, books and keywords are kind of ambiguous. 

To get back to the main question: The plan types were meant to allow you to plan different meals like Breakfast, Dinner, etc.. I personally dont use that feature and just have a single plan type called Meal, but i added it for people wo like the micro management.
As for books, if you have any ideas on how to improve this feel free to open an issue or if you want submit a PR to the docs to make them more easily understandable.

**Reply by V1d1o7** [reply]
I have actually found a good use for the multiple plan types. For me I only plan dinner so the top is the dinner for the day, below that is notes so if I need to buy anything it's noted, then the third is schedules. ![IMG_20210214_112211.png](https://user-images.githubusercontent.com/56764154/107885341-e5985100-6eb6-11eb-879a-13cbc2886379.png)

As much as I absolutely LOVE this software I very well might do that. I have no problem helping out a little bit. I need to spin up a dev branch to see if fractions show there as my beta branch doesn't, but for now I am very happy with what I can do with this.

**Reply by vabene1111** [reply]
yea beta is a little outdated as the build fails for some reason and i havn't gotten around to fixing it. The full release though is planned to be very soon (as soon as i fix a few bugs i found while running dev on my instance)
